src/ui/layout/ApplicationLayout.js

```diff
@@ -88,9 +88,9 @@ export default withStyles(styles)(function ApplicationLayout(props) {
         </div>
     );
 
-    const startsWithString = 'https://inforos-test.cern.ch';
     const isInsideIframe = window.self !== window.top;
-    const showTopBar = !(document.referrer.startsWith(startsWithString) && isInsideIframe);
+    const isInsideAllowedURL = document.referrer.match(applicationData.EL_IFURL);
```

Comment:
https://iamnotallowedtohidetheheader.com?param=https://eamlight.cern.ch

The previous URL is an exploit of the solution